### ISSUE TITLE
Feature Request: Just return JSON

### ISSUE THREAD

**Issue by joerussbowman** [starfishmod/jquery-oembed-all#38]
Something that would make jquery.oembed easier to fit into existing projects would be to just return json., which developers could then pass to something like underscore templates. This would make it a better fit for working with libraries like backbone and give developers a better format option for display

**Comment by param-veer** [comment]
hello joerussbowman  thanks for taking out time to work with this plugin, from your post i came to know u made some tweaks for this plugin, wanted to ask if u could share it , i am already running this wonderful plugin forked by starfishmod , i am using this in a live website , can you share it ?

**Comment by joerussbowman** [comment]
Most of my tweaks are on my branch here? - https://github.com/joerussbowman/jquery-oembed-all

At this point I'm pretty far into modding it specifically fit to unscatter.com, so not sure how much sense it would be to update that repo with those. I'd like to eventually work on this idea, but right now am focused on the fact my site doesn't render correctly in ie at all, so am doing a complete rewrite of the front end.

**Comment by param-veer** [comment]
i can very well understand . i wish i could help you writing this as i also expressed this to starfishmod but really i dnt have any idea about this . i wrote a plugin for atmost 6 websites in php for embed with custom height and width , but his yql,open graph etc never worked on these . plus i have a live website running which gives a lot of console error using the present starfishmod plugin. But it surly works . i really wanted a plugin for embeds as its most required at this time for my website. i request you if you could take some time out in near by days for this plugin . I m using your version from now . thank you again for forking this and making it more flexible for better embeds :)

**Comment by joerussbowman** [comment]
Sorry I do this stuff as a hobby and don't put any time to it unless I have it. I would suggest maybe hiring a developer if you need someone to do the updates you require on a schedule.

**Comment by catherinek** [comment]
+1

**Comment by detj** [comment]
yes, just JSON makes a lot of sense

**Comment by DoobleD** [comment]
I agree it'd be really a great thing, allowing us to easily customs our rendering.
